Sources/MapboxNavigation/CameraController.swift

```diff
@@ -101,7 +101,9 @@ class CameraController: NavigationComponent, NavigationComponentDelegate {
         case .transitionToFollowing, .following:
             navigationViewData.navigationView.overviewButton.isHidden = false
             navigationViewData.navigationView.resumeButton.isHidden = true
-            navigationViewData.navigationView.wayNameView.isHidden = false
+            if let roadName = navigationViewData.navigationView.wayNameView.text {
```

Comment:
`wayNameView` also has `attributedText` property, but looks like `UILabel` always keeps them in sync.

Reply:
Yes. The two properties when change one, the other one will be replaced too.